src/ui/dialogs/avatarupload.ts

```diff
@@ -0,0 +1,177 @@
+import Dialog from '../Dialog'
+import Client from '../../Client'
+
+let avatarUploadTemplate = require('../../../template/avatarUploadTemplate.hbs');
+
+let dialog: Dialog;
+let base64data : string;
+let mimetype : string;
+
+let height: string;
+let width: string;
+
+let account;
+
+export default function() {
+
+   let content = avatarUploadTemplate();
+
+   dialog = new Dialog(content);
+   let dom = dialog.open();
+
+   let accounts = Client.getAccountManager().getAccounts();
+   for (let i=0;i<accounts.length;i++){
+       let accountoption = $('<option value="'+i+'">'+accounts[i].getUid()+'</option>');
+       dialog.getDom().find('select[name="account"]').append(accountoption);
+   }
+
+   dialog.getDom().find('select[name="account"]').change((e)=>{
+       let val = Number($(e.target).val());
```

Comment:
I think `parseInt(val, 10)` would be better.

Reply:
okay...